app/views/layouts/_skip_links.html.slim

```diff
@@ -1,4 +1,2 @@
-- if show_cookies_banner?
-  = govuk_skip_link(href: "#cookie-consent", text: t("app.skip_link.cookie"))
```

Comment:
If not used anywhere else, could we remove the locale definition? `"app.skip_link.cookie"`